chain/consensus/mir/mine.go

```diff
@@ -128,21 +129,21 @@ func Mine(ctx context.Context, addr address.Address, api v1api.FullNode) error {
 			log.With("epoch", nextEpoch).
 				Infof("retrieved %d crossmsgs from mempool", len(crossMsgs))
 
-			var refs []*RequestRef
+			var requests []*mirRequest.Request
 
-			refs, err = m.AddSignedMessages(refs, msgs)
+			requests, err = m.AddSignedMessages(requests, msgs)
 			if err != nil {
 				log.With("epoch", nextEpoch).
 					Errorw("unable to push messages", "error", err)
 			}
 
-			refs, err = m.AddCrossMessages(refs, crossMsgs)
+			requests, err = m.AddCrossMessages(requests, crossMsgs)
 			if err != nil {
 				log.With("epoch", nextEpoch).
 					Errorw("unable to push cross-messages", "error", err)
 			}
 
-			m.SubmitRequests(ctx, refs)
```

Comment:
I realized that this is probably called a lot, as it is in the `default` branch of the `select` statement. It looks like it creates a sort of busy-wait loop, always burning up one CPU no matter what.
It's not a big deal for now though, as I expect this branch to be part of the request pool (mempool wrapper) module, where it we'll take this into account.

This is only a comment, no direct action required, so pls mark it directly as resolved yourself when you read it.